packages/js/src/plugins/auctionHouseModule/operations/getBuyerBalance.ts

```diff
@@ -32,7 +41,9 @@ export type GetBuyerBalanceOperation = Operation<
  * @category Inputs
  */
 export type GetBuyerBalanceInput = {
+  /** The Auction House in which to get buyer's escrow balance. */
   auctionHouse: Pick<AuctionHouse, 'address'>;
```

Comment:
Since we only need the address here let's just require the Public Key directly.

```suggestion
  auctionHouse: PublicKey;
```